Forms/MainForm.cs

```diff
@@ -75,12 +78,22 @@ private void UpdateConfigurationSubstitutions()
         {
             Configuration.DefaultSubstitutions.Clear();
             Configuration.DefaultSubstitutions.AddRange(substitutionList1.Substitutions);
+            LoadConfigurationDetails();
         }
 
 
         private void LoadConfigurationDetails()
         {
-            txtWordList.Text = Configuration.DefaultWords.ToString();
+            string wordtext = Configuration.DefaultWords.ToString();
+            if (txtWordList.Text != null && txtWordList.Text != wordtext && txtWordList.Text != wordtext + " ")

```

Comment:
> But that will still leave the problem of erasing trailing spaces. That's
caused by my removing empty items when parsing the list into words. I think
that's a good thing but it means the conversion back to text for the box
will never include trailing spaces.

Using the keyDown event, we can prevent the user from typing a space next to another space

> Actually the only time it should need ever need updating via code is during the initial load, come to think of it. 

I'm also thinking about when we will have #6; but we should probably wait then...  

I also think a timer-based approach is a good idea, it might be more work though, if you don't feel like going down that path now, I would at least try to handle space input issues...